fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/ValueIndexLikeMatchCandidate.java

```diff
@@ -99,7 +99,7 @@ default List<MatchedOrderingPart> computeMatchedOrderingParts(@Nonnull MatchInfo
             builder.add(
                     MatchedOrderingPart.of(value,
                             comparisonRange == null ? ComparisonRange.Type.EMPTY : comparisonRange.getRangeType(),
-                            queryPredicate,
+                            (comparisonRange == null || comparisonRange.isEmpty()) ? null : queryPredicate,
```

Comment:
Not sure what that does.

Reply:
It is a left-over from a previous iteration ... removed it, thanks for spotting it.